doc/doxygen/headers/automatic_and_symbolic_differentiation.h

```diff
@@ -14,47 +14,395 @@
+ *    derivatives of the function then can be computed by referring to the code path tracked and stored on
+ *    the tape.
+ *    (This could perhaps be achieved, for example, through evaluation of the function around the point
+ *    of interest.) There exist strategies to deal with situations where the taped function is not
+ *    smooth at the evaluated point, or if it is not analytic. Furthermore, one might need to consider the
+ *    case of branched functions, where the tape is no longer sequential, but rather forks off on a different
+ *    evaluation path to that due to the original recorded inputs.
+ * -# Methods based on <a href="https://en.wikipedia.org/wiki/Expression_templates">expression templates</a>
+ *    leverage the computational graph
+ *    (in this case, a <a href="https://en.wikipedia.org/wiki/Directed_acyclic_graph">directed acyclic graph (DAG)</a>),
+ *    constructed from the abstract syntax tree (AST), that resolves the function output from its input values.
+ *    The outermost leaves on the tree represent the independent variables or constants, and are transformed by unary
+ *    operators and connected by binary operators (in the most simple case). Therefore, the operations performed on
+ *    the function inputs is known at compile time, and with that the associated derivative operation can also be defined
+ *    at the same time. The compiled output type returned by this operator need not be generic, but can rather be
+ *    specialized based on the specific inputs (possibly carrying a differential history) given to that specific
+ *    operator on the vertex of the DAG. In this way, a compile-time optimized set of instructions can be generated
+ *    for the very specialized individual operations used to evaluate each intermediate result of the dependent
+ *    function.
+ *
+ * Each of these methods, of course, has its advantages and disadvantages, and one may be more appropriate
+ * than another for a given problem that is to be solved. As the aforemetioned implementational details
+ * (and others not discussed) may be hidden from the user, it may still be important to understand the
+ * implications, run-time cost,  and potential limitations, of using any one of these "black-box"
+ * auto-differentiable numbers.
+ *
+ * In addition to the supplied linked articles, resources used to furnish the details supplied here include:
+ *
+ * @code{.bib}
+ * @InProceedings{Fike2011a,
+ *   author    = {Fike, Jeffrey A and Alonso, Juan J},
+ *   title     = {The Development of Hyper-Dual Numbers for Exact Second-Derivative Calculations},
+ *   booktitle = {49th {AIAA} Aerospace Sciences Meeting including the New Horizons Forum and Aerospace Exposition},
+ *   year      = {2011},
+ *   volume    = {886},
+ *   pages     = {124},
+ *   month     = {jan},
+ *   publisher = {American Institute of Aeronautics and Astronautics},
+ *   doi       = {10.2514/6.2011-886},
+ * }
+ * @endcode
```

Comment:
you can keep the two inside a single `@code{.bib}`

Reply:
I showed @davydden that this currently renders each reference within its own "code block". I prefer this style, and he's agreed that I need not change it.